explore/src/main/scala/explore/tabs/ObsTabContents.scala

```diff
@@ -255,24 +273,27 @@ object ObsTabContents {
             ),
             <.div(
               ^.key := "target",
-              Tile("Target")(
-                LiveQueryRenderMod[ObservationDB,
-                                   TargetEditQuery.Data,
-                                   Option[TargetEditQuery.Data.Target]
-                ](
-                  TargetEditQuery.query(targetId),
-                  _.target,
-                  NonEmptyList.of(TargetEditSubscription.subscribe[IO](targetId))
-                ) { targetOpt =>
-                  <.div(
-                    <.div(
-                      (props.userId.get, targetOpt.get).mapN { case (uid, _) =>
-                        val stateView = ViewF.fromState[IO]($).zoom(State.options)
-                        TargetBody(uid, targetId, targetOpt.zoom(_.get)(f => _.map(f)), stateView)
+              <.div(
+                ^.key := s"target-$targetId",
+                Tile("Target")(
```

Comment:
You can do `Tile("Target").withKey(s"target-$targetId")(` to avoid the extra `div`. Same applies to the "Notes" `Tile` above.

Also, can we remove the outer `div`? (the one with `key := "target"`)

Reply:
Nice tip, we can't remove the outer one as react grid layout needs a fixed id

Reply:
Maybe LiveRenderMod should take a key?

Reply:
It does, since it uses the same mechanism in `ReactProps`. However, the `.withKey` has to go after the render function. Eg:

``` scala
  LiveQueryRenderMod[ObservationDB,
                                     TargetEditQuery.Data,
                                     Option[TargetEditQuery.Data.Target]
                  ](
                    TargetEditQuery.query(targetId),
                    _.target,
                    NonEmptyList.of(TargetEditSubscription.subscribe[IO](targetId))
                  ) { targetOpt =>
                       ...
                  }.withKey("key")
```

Reply:
This works fine on the `LiveQueryRenderMod` element but I'm having troubles with the one on `Tile`. I'll address that in another PR